app/javascript/react/screens/App/App.scss

```diff
@@ -1 +1,3 @@
 @import './Overview/Overview.scss';
+@import './Overview/screens/MappingWizard/components/MappingWizardClustersStep/MappingWizardClustersStep.scss';
```

Comment:
one thing i did wrong before probably... would be nice to localize the `@imports` a bit further...

`@import ./Overview/Overview.scss`

Overview.scss then imports:
`@import ./screens/MappingWizard.scss`

MappingWizard.scss then imports:
`@import ./components/MappingWizardClustersStep/MappingWizardClustersStep.scss`

WDYT?

Reply:
I was thinking the same but wasn't sure what our convention was going to be, so I just left it lol.  Makes sense to me!